### ISSUE TITLE
As a user I want to see my dashboard

### ISSUE THREAD

**Issue by lcabrini** [borderitsolutions/amadaa#72]

**Comment by lcabrini** [comment]
This is basic functionality that must be there. I propose this issue to be added to the product backlog. @BorderITS

**Comment by BorderITS** [comment]
Agreed

**Comment by lcabrini** [comment]
Next I want to propose this issue to be added to the sprint backlog.

**Comment by BorderITS** [comment]
Agreed

**Comment by BorderITS** [comment]
Users can now see their dashboard and propose we also move this to testing

**Comment by lcabrini** [comment]
Yes, I'm also thinking about that. But here I am a little confused. We have to clarify something.

The functionality stated in the user story is done. However, the dependent UI tasks are not. So here is where I'm not sure if we should:

1. move the story to testing. That means the story itself is working. UI can be updated later; or
2. we keep it in implementation until the UI is done for it.

At first, I thought (1) would be the best, but now I'm leaning to (2). Functionally, the user can see their dashboard. There is not much on it (that should be the topic of other user stories) and it doesn't look very good (that is the designers' headache).

So, what do you think? Do we move it or wait for the designers?

**Comment by BorderITS** [comment]
I think we should wait because we need to follow procedure

**Comment by BorderITS** [comment]
your pull request gives me error

**Comment by lcabrini** [comment]
But this issue has nothing to do with my pull request.
